src/Data/Functor/Interval.hs

```diff
@@ -78,6 +81,21 @@ instance Show (f a) => Show (Interval f a) where
   showsPrec p i = showParen (p > 3) $ showsPrec 4 (inf i) . showString "..." . showsPrec 4 (sup i)
   {-# INLINE showsPrec #-}
 
+instance Show1 f => Show1 (Interval f) where
+  liftShowsPrec = gliftShowsPrec
```

Comment:
In addition to not wanting to take on the extra dependency, it strikes me as strange that we’d have different behaviour for `Show1` and `Show`.